### ISSUE TITLE
PyCharm type warning: Window doesn't implement all abstract methods

### ISSUE THREAD

**Issue by JackAshwell11** [pythonarcade/arcade#1131]
## Bug Report

PyCharm returns the following warning when subclassing arcade.Window:

`Class Window must implement all abstract methods`.

The specific abstract method is set_mouse_platform_visible in pyglet's BaseWindow class.py and is currently commented out in https://github.com/pythonarcade/arcade/blob/development/arcade/application.py#L711.

### Actual behavior:

PyCharm displays a warning. I'm not sure about other IDEs.

### Expected behavior:

PyCharm shouldn't display an error.

### Steps to reproduce/example code:

```py
import arcade


class Window(arcade.Window):
    def __init__(self):
        super().__init__()


window = Window()
window.run()
```

**Comment by einarf** [comment]
This is a window specific issue becaue pyglet's win32 window don't implement this method. We should just bring back the wrapper function for now and make a pull request fixing this in pyglet.

**Comment by einarf** [comment]
The existing wrapper function was removed because it was confusing in docs, but it turns out it was overridden for a good reason. Beginners will see errors in code inspection tools for example in pycharm. We should wrap this function again with better docstrings and remove it after new pyglet release.

**Comment by einarf** [comment]
![bilde](https://user-images.githubusercontent.com/5418180/160193959-582ed0f6-de3a-41a2-ad11-179fad6a56e8.png)

**Comment by einarf** [comment]
Temp fix 5caae3fe4177849da1fd835a1155407fc3a85b1c

**Comment by einarf** [comment]
Added a wrapper method in 2.6.13, but we still need to figure out if this can be fixed in pyglet.
